test/unit/testBraveAlerter/testBraveAlerterSendReminderMessageForSession.js

```diff
@@ -287,6 +295,10 @@ describe('braveAlerter.js unit tests: sendReminderMessageForSession', () => {
         alertType: ALERT_TYPE.BUTTONS_NOT_URGENT,
         deviceName: 'Bathroom 2',
         reminderMessage: 'My message',
+        language: 'en',
+        t: function mock_t(key, options) {
+          return `${key} - ${options.lng}`
+        },
```

Comment:
For my own understanding - do you mind explaining why you've added the `language` and `t` function for this specific test case? I don't think I see where it's been used. 

If it's not being used in this test (and you've added these keys so that the data is consistent), that's totally fine but I would suggest making sure that all the `this.braveAlerter.sendReminderMessageForSession` functions include `t` and `language` then.

Reply:
@ahmaddaadaa , agree with Tara here. You've included `language` and `t` for lines 48, 284, but not 87, 126, 169, 203, 240, and 320.

Reply:
language and t function are now part of the alertInfo object that is passed to the Brave Alert library. Even though they are not being used in this test, I added them for consistency